pkg/accounts/kubernetes/validator.go

```diff
@@ -32,21 +36,49 @@ type kubernetesAccountValidator struct {
 }
 
 func (k *kubernetesAccountValidator) Validate(spinSvc v1alpha2.SpinnakerServiceInterface, c client.Client, ctx context.Context, log logr.Logger) error {
-	if err := k.validateSettings(ctx, log); err != nil {
+	err := k.validateSettings(ctx, log)
```

Comment:
wondering why this was changed. I generally find better (and more readable but I guess that's subjective) to do:
```
if err := xxx; err != nil {
  return err
}
```
as it doesn't introduce another scope variable - even when another variable is done further down.

Reply:
The reason is that further down I'm trying to reassign `spinSvc` the output of a function that returns an error, so I needed to reuse this same `err` to not introduce any new variables.